src/App.jsx

```diff
@@ -19,28 +19,28 @@ export default function App() {
       <Link to="/HomePage">
         <button type="button">This is a Link to the Home page!</button>
       </Link>
-      <Link to="/PackDetails">
+      <Link to=':packid/PackDetails'>
```

Comment:
apparently the eslinter does not like single quotes on the Link. please change this back to double quotes!